File: 1. Conditional Problems/Conditional Problems in Python/p4.py
# write a python program to find maximum between three numbers

a,b,c = map(int,input("Enter a , b , c: ").split())

if a>b and a >c:
    print(f"{a} is highest number")
elif b >a and b>c:
    print(f"{b} is highest number")
elif c >a and c>b:
    print(f"{c} is highest number")
else:
    print("You have given multple numbers that have the same value.")

File: 1. Conditional Problems/Condional Problems in C/p4.c
// write a C program to find maximum between three numbers

#include <stdio.h>

int main()
{
    int a,b,c;
    printf("Enter a , b , c: ");
    scanf("%d%d%d",&a,&b,&c);

    if (a >= b && a >= c) {
        printf("%d is the highest",a);
    }
    else if (b>=c) {
        printf("%d is the highest",b);
    }
    else {
        printf("%d is the highest",c);
   
    }
    return 0;

}